rotkehlchen/db/upgrades/v30_v31.py

```diff
@@ -26,6 +29,12 @@ def upgrade_v30_to_v31(db: 'DBHandler') -> None:
         PRIMARY KEY (type, identifier)
         );
         """)
+
+    # Delete kraken trades so they can be requeried
+    cursor.execute('DELETE FROM trades WHERE location="B";')
+    cursor.execute('DELETE FROM used_query_ranges WHERE name LIKE "kraken_trades_%";')
```

Comment:
The test `rotkehlchen/tests/data_migrations/test_migrations.py::test_migration_1` is failing because the data migrations are executed after db migrations and this lines deletes data that the test expects to be there before

Reply:
I changed that so thatthe migrations test runs without any DB upgrades running on the tested DB. Which is imo the right way to test it, as it's testing the migration as it was intended to run.

Reply:
Better than my patch then